### ISSUE TITLE
feat(docs): one click "add example" button

### ISSUE THREAD

**Issue by mesqueeb** [millsp/ts-toolbelt#144]
#### Teachability, Documentation, Adoption, Migration Strategy

In the documentation, there is often a case where I find the example lacking and would like to suggest a new example or add one where there is none.

Currently there's a link like so:
<img width="455" alt="image" src="https://user-images.githubusercontent.com/3253920/93073412-dae64f80-f6bd-11ea-9270-c4eaaad83215.png">

However, the **problem is** that this opens the page at a specific blob like this link:
https://github.com/millsp/ts-toolbelt/blob/3859d18/src/Any/Compute.ts#L48

From here it's pretty complicated how I can easily make a change and open a PR.

With other sites that use eg. Vue Press, you'll get this "edit these docs" link like so:
<hr />
<img width="693" alt="Screenshot 2020-09-14 at 19 12 47" src="https://user-images.githubusercontent.com/3253920/93073752-5e07a580-f6be-11ea-9c89-9023a1ba1deb.png">
<hr />
Which, if you don't have a fork, will automatically create a fork and prompt to open a PR after a change.

Maybe you could slightly make this adjustment to the links you added to your examples so it does the same?

**Comment by millsp** [comment]
Definitely! I was needing to move the website to `.md` files but I got too busy lately. I though I'd move to `docz` but I'm not sure if they have such a feature for editing. Don't hesitate to share a list of doc generators you like.

**Comment by millsp** [comment]
Thanks for your help

**Comment by mesqueeb** [comment]
Well, I like Vue Press, but I know you're a React guy, so we're kinda in different camps here 😛

**Comment by millsp** [comment]
Yeah, for us react guys there's docusaurus :t-rex:
